apps/emqx_durable_storage/test/emqx_ds_storage_layout_SUITE.erl

```diff
@@ -292,6 +299,28 @@ t_replay(Config) ->
     ?assert(check(?SHARD, <<"#">>, 0, Messages)),
     ok.
 
+t_replay_sys(_Config) ->
```

Comment:
What property does this testcase try to verify? Why do I need to care about this property? Please add a comment answering these questions.